### PR TITLE
Add CLI Options #19

### PR DESCRIPTION
**Pull request by cnwilkin**
## Adding additional parameters
- [x] build.config location
- [x] host
- [x] port
- [x] .repositories path
- [x] .ravendb path
- [x] Raven connection string if you don't want to use local, embedded Raven

### CODE REVIEW

**Comment by cnwilkin** [comment]
This is ready for review

File: qed/Functions/CreateRavenStore.cs [hunk 1]
```diff
@@ -11,10 +12,16 @@ public static IDocumentStore GetRavenStore()
             if (ravenStore != null)
                 return ravenStore;
 
-            ravenStore = new EmbeddableDocumentStore
-            {
-                DataDirectory = "~\\.ravendb"
-            };
+            if (string.IsNullOrEmpty(GetConfiguration<string>(Constants.Configuration.RavenConnectionStringKey)))
```
Comment (half-ogre) [inline qed/Functions/CreateRavenStore.cs:15]: I'm not a fan of needless braces, but I think wrapping the `ravenStore` setting in the `if` and `else` is a bit more readable. Could make that change please?

**Comment by half-ogre** [comment]
Great stuff @cnwilkin! There's one minor style thing I'd like to see fixed before this is merged.

File: qed/Functions/CreateRavenStore.cs (hunk 1 above)
Comment (cnwilkin) [inline qed/Functions/CreateRavenStore.cs:15]: Sorry I'm not entirely sure what your looking for are you just wanting to add `{ }`? Like

```
if (string.IsNullOrEmpty(GetConfiguration<string>(Constants.Configuration.RavenConnectionStringKey)))
{
    ravenStore = new EmbeddableDocumentStore
    {
        DataDirectory = GetConfiguration<string>(Constants.Configuration.RavenDataDirectoryKey)
    };
}
....
```

Comment (half-ogre) [inline qed/Functions/CreateRavenStore.cs:15]: Correct; sorry I wasn't clear.

**Comment by cnwilkin** [comment]
No problem got them put in.

**Comment by half-ogre** [comment]
snf03tvbiz1-280_462948a

**Pull request merged**